### PR TITLE
fix: display a human-readable error message when report is completely empty

### PR DESCRIPTION
**Pull request by shadowusr**
What it looks like:

### CODE REVIEW

File: lib/static/components/suites.jsx [hunk 1]
```diff
@@ -112,7 +112,16 @@ function Suites(props) {
...
     const {visibleRootSuiteIds} = props;
 
-    if (isEmpty(visibleRootSuiteIds)) {
+    if (props.isReportEmpty) {
+        return <div className="sections sections_type_empty">
+            There are no tests to show. Please, try the following:
+            <ul>
+                <li>Check if your project contains any tests;</li>
+                <li>Check if the tool you are using is configured correctly and is able to find your tests;</li>
+                <li>Check if some critical error has occurred in logs that prevented HTML reporter to collect any results.</li>
```
Comment (Kabir-Ivan) [inline lib/static/components/suites.jsx:121]: I would instead write:
"Check if some critical error has occurred in logs that prevented HTML reporter from collecting any results"

File: lib/static/components/suites.jsx [hunk 2]
```diff
@@ -112,7 +112,16 @@ function Suites(props) {
 
     const {visibleRootSuiteIds} = props;
 
-    if (isEmpty(visibleRootSuiteIds)) {
+    if (props.isReportEmpty) {
+        return <div className="sections sections_type_empty">
```
Comment (KuznetsovRoman) [inline lib/static/components/suites.jsx:116]: I would pull this `html` (as well as "There are no tests that match to selected filters" html) out to functions, so here it would look like:

```js
if (props.isReportEmpty) {
    return renderEmptyReport();
}

if (isEmpty(visibleRootSuiteIds)) {
  return renderEmptyVisibleRootSuiteIds(_getSelectedFiltersInfo());
}

return (
  <div className="sections">
  ...
)

**Review by KuznetsovRoman — APPROVED** [review]
We have a test `should not render List component if there are no visible root suite ids` in  `test/unit/lib/static/components/suites.jsx` (line 51-57) that checks rendered text.

Let's add the same test for this case, when report is empty

File: lib/static/components/suites.jsx (hunk 2 above)
Comment (shadowusr) [reply lib/static/components/suites.jsx:116]: IMO it only sacrifices readability without solving any actual problems. Keeping it as is for now.
